docs/en/04_Changelogs/4.9.0.md

```diff
@@ -2,13 +2,130 @@
 
 ## Overview
 
+A full list of module versions included in CMS Recipe {{ version }} is provided below. We recommend referencing recipes in your dependencies, rather than individual modules, to simplify version tracking. See [Recipes](/getting_started/).
+
+
+<details>
+<summary>Core module versions</summary>
+
+| Module | Version |
+| ------ | ------- |
+| silverstripe/admin | 1.9.0 |
+| silverstripe/asset-admin | 1.9.0 |
+| silverstripe/assets | 1.9.0 |
+| silverstripe/campaign-admin | 1.9.0 |
+| silverstripe/cms | 1.9.0 |
```

Comment:
```suggestion
| silverstripe/cms | 4.9.0 |
```

Reply:
Copied that from the RC change log, so I probably borked it there as well.